### PR TITLE
Upgrade GE plugin to 3.3.3

### PR DESCRIPTION
**Pull request by guylabs**

### CODE REVIEW

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -664,6 +667,16 @@
             <sha256 value="cdedd4655949d142a15e788dd88908845c0d856c436b28ef3974e24627eb9d77" origin="Generated by Gradle because artifact wasn't signed"/>
          </artifact>
       </component>
+      <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.3.2">
```
Comment (wolfs) [inline gradle/verification-metadata.xml:670]: I adding the 3.3.2 version necessary? Did we use that anywhere?

File: gradle/verification-metadata.xml [hunk 2]
```diff
@@ -3,6 +3,9 @@
    <configuration>
       <verify-metadata>false</verify-metadata>
       <verify-signatures>true</verify-signatures>
+      <key-servers>
+         <key-server uri="https://keys.openpgp.org"/>
```
Comment (wolfs) [inline gradle/verification-metadata.xml:7]: Why do you need to open the keyserver now?

File: gradle/verification-metadata.xml (hunk 1 above)
Comment (guylabs) [reply gradle/verification-metadata.xml:670]: We added it for consistency as `3.3.2` was released yesterday but we never integrated it into the Gradle build. Should we still remove it?

File: gradle/verification-metadata.xml (hunk 2 above)
Comment (guylabs) [reply gradle/verification-metadata.xml:7]: We got an error on dependency verification that the signing key was not found and we would need to provide the key server where the key is published.

**Comment by wolfs** [comment]
@bot-gradle test this

**Pull request merged**